core/docs/spark-qualification-tool.md

```diff
@@ -277,6 +277,7 @@ The tree structure of the output directory `${OUTPUT_FOLDER}/rapids_4_spark_qual
     ├── rapids_4_spark_qualification_output_persql.csv
     ├── rapids_4_spark_qualification_output_execs.csv
     ├── rapids_4_spark_qualification_output_stages.csv
+    ├── rapids_4_spark_qualification_output_mlFunctions.csv
```

Comment:
nit: rename from `*_mlFunctions.csv` to `*_mlfunctions.csv` to be consistent with naming of other csv files (all lowercase)